assets/dash.css

```diff
@@ -0,0 +1,60 @@
+#header {
+  background-color: #000 !important;
+}
+
+  #header a.btn {
+    color: #aaa;
+  }
+
+  #header a.btn:hover {
+    color: #fff;
+  }
+
+
+  #logo {
+    height: 52px;
```

Comment:
Visuals: The header height is a little heavy compared to other sites in the family (margin box height for GLAM is 48, Glean Dictionary 67 - this (coupled with h1 and padding) puts us at 104. With widescreen monitors, we can slim this down to focus more on the index content.